spacy_llm/tests/tasks/test_entity_linker.py

```diff
@@ -0,0 +1,631 @@
+# ruff: noqa: W291
+import csv
+import functools
+from pathlib import Path
+
+import numpy
+import pytest
+import spacy
+from confection import Config
+from spacy import Vocab
+from spacy.kb import InMemoryLookupKB
+from spacy.tokens import Span
+from spacy.training import Example
+from spacy.util import make_tempdir
+
+from spacy_llm.registry import fewshot_reader, file_reader
+from spacy_llm.tasks.entity_linker import EntityLinkerTask, make_entitylinker_task
+from spacy_llm.tasks.entity_linker.util import SpaCyPipelineCandidateSelector
+from spacy_llm.util import assemble_from_config
+
+from ..compat import has_openai_key
+
+EXAMPLES_DIR = Path(__file__).parent / "examples"
+TEMPLATES_DIR = Path(__file__).parent / "templates"
+
+
+@functools.lru_cache()
+def build_el_pipeline(nlp_path: Path, desc_path: Path) -> None:
+    """Builds and persists pipeline with untrained EL component and initialized toy knowledge base.
+    nlp_path (Path): Path to store pipeline under.
+    descriptions_path (Path): Path to store descriptions file under.
+    """
+    nlp = spacy.load("en_core_web_md")
+    nlp.add_pipe("entity_linker")
+    kb = InMemoryLookupKB(
+        vocab=nlp.vocab, entity_vector_length=nlp.vocab.vectors_length
+    )
+
+    # Define entities (move to file?).
```

Comment:
Maybe for the example use-case, it'd be nicer to have this in some sort of "kb" file that's read & parsed, yea.

Reply:
Done in https://github.com/explosion/spacy-llm/pull/241/commits/0dba15cd929d781c795dbb4ab4d7cd3b0019cb5c.